spark-jobs/src/main/scala/filodb/downsampler/index/DSIndexJobMain.scala

```diff
@@ -7,13 +7,14 @@ import org.apache.spark.sql.SparkSession
 
 object DSIndexJobMain extends App {
   import DSIndexJobSettings._
-  val iu = new IndexJobDriver(hour() - 1) // Run for the previous hour
+  val migrateUpto = hour() - 1
+  val iu = new IndexJobDriver(migrateUpto - batchLookbackInHours, migrateUpto) //migrate partkeys between these hours
   val sparkConf = new SparkConf(loadDefaults = true)
   iu.run(sparkConf)
   iu.shutdown()
 }
 
-class IndexJobDriver(epochHour: Long) extends StrictLogging {
+class IndexJobDriver(from: Long, to: Long) extends StrictLogging {
```

Comment:
document whether from and to are inclusive.

Reply:
rename to `fromHour` and `toHour`

Reply:
done, updated with comments